src/test/java/teammates/test/cases/webapi/SearchStudentsActionTest.java

```diff
@@ -1,6 +1,6 @@
 package teammates.test.cases.webapi;
 
-import org.junit.Test;
+import org.testng.annotations.Test;
```

Comment:
Normally I would say that this shouldn't be included as it is not in the scope of the issue, but I will close one eye this time.